src/main/java/liuni/health/LiUniHealthCheck.java

```diff
@@ -3,11 +3,8 @@
 import com.codahale.metrics.health.HealthCheck;
 
 public class LiUniHealthCheck extends HealthCheck {
-
-    public LiUniHealthCheck() { }
-
     @Override
-    protected Result check() {
+    public Result check() {
```

Comment:
this shouldn't need to be updated as long as your unit test is in the same package.